benchmarks-mpp/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
This file is unneeded because there's no standalone wrapper in this directory.

Reply:
deleted